wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
+1. **Glissez et déposez les deux icônes dans le dossier Applications** pour installer Jamulus.
+1. **Executez Jamulus**. Vous devriez maintenant pouvoir utiliser Jamulus comme n'importe quelle autre application. Si vous voyez un message qui vous dit que Jamulus ne peut pas être ouvert parce que le développeur ne peut pas être vérifié, lisez la suite et jetez un coup d'oeil à la [section suivante](#jamulus-ne-peut-pas-être-ouvert-car-le-développeur-ne-peut-pas-être-vérifié).
+
+_Vous pouvez supprimer le dossier dans le répertoire Downloads contenant le `.dmg` et éjecter le lecteur "Jamulus" de votre bureau. Ils ne sont plus nécessaires._
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous en avez envie.
+
+## "Jamulus" ne peut pas être ouvert car le développeur ne peut pas être vérifié
+
+**La première fois que vous essaierez de double-cliquer sur Jamulus pour l'exécuter, vous verrez probablement un message disant qu'il ne peut pas être ouvert.**
+
+Pour résoudre ce problème, accédez au dossier Applications via le Finder, cliquez sur Jamulus, faites un clic droit (ou un contrôle-clic) et sélectionnez « _Ouvrir_ » dans le haut du menu contextuel.
+
+Vous aurrez alors une version légèrement différente du même message, qui vous permettra de cliquer sur « _Ouvrir_ » et de continuer à l'exécuter. Cet avertissement ne s'affichera plus - il suffit de double-cliquer dessus pour le lancer. Pour plus d'informations, consultez [cette page d'assistance Apple](https://support.apple.com/fr-fr/guide/mac-help/mh40616/mac){: target="_blank" rel="noopener noreferrer"}.
+
+## Tout est installé ?
+
+**Jamulus** a été installé et peut être utilisé maintenant. Si vous n'avez pas encore installé et configuré votre matériel audio, consultez la section [configuration du matériel](Hardware-Setup).
+
+Pour des informations plus détaillées sur l'utilisation de Jamulus, vous pouvez aussi consulter l'[aide de Jamulus (Maunuel du Logiciel)](Software-Manual).
+
```

Comment:
We've call it "_Manuel du logiciel_", not "_aide de Jamulus (Maunuel du Logiciel)_" ([see here](https://github.com/jamulussoftware/jamuluswebsite/blob/translation-3.6.1/_data/fr/navigation.yml)). We need to keep the consistency across all the translation.

_(by the way, there is a typo: Ma**u**nuel)_

Reply:
Yes but the title of the page in english is « Software Help (User Manual) » wich I've translated as « Aide de Jamulus (Manuel du logiciel) » (instead of « Manuel de l'utilisateur ».

So If we change here…

Reply:
I saw you changed this, but the typo I referenced in the message above: Ma**u**nuel/Manuel is still present. Please fix it.

Also, there is no need for capital letter, "manuel du logiciel" is fine.

Reply:
I missed this, fixed.

Reply:
Thanks.